CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Maintainers
+This repo is currently maintained by Ken Liao (@yknl). 
+
+# Pull Request Reviewers
+Hank Stoever (@hstove)
```

Comment:
If you add all three here, the result may be they show up on every review.  You can require a number of reviewers:

https://help.github.com/articles/about-pull-request-reviews/#required-reviews

The alternative is to indicate which of these people should review.